metadata.yaml

```diff
@@ -45,6 +47,9 @@ storage:
   certs:
     type: filesystem
     minimum-size: 50M
```

Comment:
And isn't 50M for certs "a bit" too much?

Reply:
True, 1M would be more than enough.

Reply:
I modified the sizes, in the case of tmp it's really hard to estimate, we can't know how big a vault backend can get in a production deployment.
If you have other suggestions let me know.
I modified the sizes of the other volumes too, let me know if I should remove it from this pr.